### PR TITLE
TDR 3210/3249 Standard guide and faq cotent changes

### PR DESCRIPTION
**Pull request by thanhz**
also includes content change from TDR-3212

### CODE REVIEW

File: app/views/help.scala.html [hunk 1]
```diff
@@ -371,29 +362,34 @@ <h3 class="govuk-heading-s" id="file-checks">Step 4: File checks</h3>
...
+                If it has been agreed, you may proceed to add information to the following fields:
             </p>
             <ul class="govuk-list govuk-list--bullet">
-                <li>FOI decision asserted – this is the date of the Advisory Council approval</li>
-                <li>Closure start date – this is the date of the last meaningful change to the record</li>
-                <li>Closure period – this is the number of years the record is closed from the closure start date</li>
-                <li>FOI exemption code – you should search and select at least one exemption code per file</li>
-                <li>Title sensitivity – you should confirm if the title is sensitive to the public. If it is, you can add an alternative title</li>
-                <li>Description Sensitivity – you can only provide an alternative description if you have set a description on the Descriptive Metadata page first</li>
+                <li>FOI decision asserted – you should enter the date that the closure schedule was agreed with the Advisory Council and/or the National Archives. </li>
```
Comment (TomJKing) [inline app/views/help.scala.html:386]: Unnecessary space before the closing tag `</li>`

File: app/views/help.scala.html [hunk 2]
```diff
@@ -371,29 +362,34 @@ <h3 class="govuk-heading-s" id="file-checks">Step 4: File checks</h3>
...
+                <li>Closure start date – you should enter the date of the last meaningful change to the record.</li>
+                <li>Closure period – you should enter the number of years the record must be closed from the closure start date.</li>
+                <li>FOI exemption code – you should search and select at least one exemption code per record</li>
```
Comment (TomJKing) [inline app/views/help.scala.html:389]: Missing full stop at end of list item: `... code per record`

File: app/views/help.scala.html [hunk 3]
```diff
@@ -371,29 +362,34 @@ <h3 class="govuk-heading-s" id="file-checks">Step 4: File checks</h3>
...
+                <li>Is the title sensitive for the public? – under this question, you should confirm if the title is sensitive to the public. If it is, you can add an alternative title.</li>
+                <li>Is the description sensitive for the public? – under this question, you should confirm if the description is sensitive to the public. If it is, you can add an alternative description.
+                    You can only provide an alternative description if you have already added a description on the Descriptive Metadata page.</li>
```
Comment (TomJKing) [inline app/views/help.scala.html:392]: Should there be a line break between the sentences?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of very minor editorial comments, also:

- Has the QR code in the new screenshot been made invalid?
- Should the `the` in `the National Archives` be capitalised? On some of the pages it is, but not in the copy changes. Probably should check with someone.

File: app/views/help.scala.html (hunk 3 above)
Comment (thanhz) [reply app/views/help.scala.html]: It doesn't affect the way it's displayed on the page but I've removed the line break anyway

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**